ui.tests/test-module/specs/wizard/wizard.authoring.spec.js

```diff
@@ -155,4 +163,40 @@ describe('Page - Authoring', function () {
         });
 
     });
+
+    context('Test Wizard Component String Language', function (){
+        beforeEach(function () {
+            // this is done since cypress session results in 403 sometimes
+            cy.visit('http://localhost:4502/aem/start.html');
+            cy.get('#username').type('admin');
+            cy.get('#password').type('admin');
+            cy.get('#submit-button').click();
+            cy.on('uncaught:exception', () => {
+                return false
+            });
+        });
+        it('Wizard String language test', function ()  {
+            changeLanguage("de");
+            cy.visit('http://localhost:4502/libs/fd/fm/gui/content/forms/af/create.html').then(() => {
+                // Your tests here
+                cy.wait(30000);
+                cy.get('[data-item-id="/conf/core-components-examples/settings/wcm/templates/af-blank-v2"]').click();
+                cy.get('.spectrum-Button_e2d99e.spectrum-Button--cta_e2d99e').click();
+                cy.get('input[name="submitDialogTitle"]').type('language-test');
+                cy.get('.spectrum-Button_e2d99e.spectrum-Button--cta_e2d99e.spectrum-ButtonGroup-Button_b10c2f').click();
+            });
+
+            changeLanguage("en");
+            cy.visit('http://localhost:4502/editor.html/content/forms/af/language-test.html');
+            const dataPath = "/content/forms/af/language-test/jcr:content/guideContainer/*",
+                responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+            cy.selectLayer("Edit");
+            cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Wizard Layout", afConstants.components.forms.resourceType.wizard);
+            cy.get('body').click(0, 0);
+            cy.get('[data-path="/content/forms/af/language-test/jcr:content/guideContainer/wizard/*"]').invoke('attr', 'data-text').should('equal', 'Please drag Wizard components here');
```

Comment:
this is only testing for english locale, can you add one more locale.
Rest of PR looks good to me.

Reply:
Added test case for another language